code/modules/cargo/packs.dm

```diff
@@ -1695,6 +1744,7 @@
 
 /datum/supply_pack/misc/foamforce/bonus
 	name = "Foam Force Pistols Crate"
+	desc = "psst..hey bud... remember those old foam force pistols that got discontinued for being too cool? well i got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
```

Comment:
Capitalize well, anything after a !, ?, or . Is a new sentence, so capitalize that shit.